js-miniapp-bridge/src/common-bridge.ts

```diff
@@ -227,6 +227,40 @@ export class MiniAppBridge {
       );
     });
   }
+
+  /**
+   * Associating getUsername function to MiniAppBridge object.
+   * This function returns username from the user profile
+   * (provided the rakuten.miniapp.user.USER_NAME is allowed by the user)
+   * It returns error info if user had denied the custom permission
+   */
+  getUserName() {
+    return new Promise<string>((resolve, reject) => {
+      return this.executor.exec(
+        'getUserName',
+        null,
+        success => resolve(success),
```

Comment:
It will return username directly so I think we can replace `success` with `username`. The same for `getProfilePhoto()`.

Reply:
Done